openshift-ci/Dockerfile.metadata

```diff
@@ -0,0 +1,17 @@
+# see: https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md
+FROM scratch
+
+ARG CHANNEL=4.6
```

Comment:
Will we need to update the dockerfile for each version?

Reply:
the idea was not to, that's because I used ARG.